### DISCUSSION TITLE
Support for Client Certificates in LoadTestToolbox

### DISCUSSION THREAD

**Discussion by twoday-TommyChristiansen** [ecoAPM/LoadTestToolbox#111] [Q&A]
Hi
I'm looking to tryout LoadTestToolbox for some load testing scenarios and was wondering if there is support for client certificates?
Our testing environment requires client certificate authentication, and I couldn't find any documentation related to this feature.

Could you please let me know if LoadTestToolbox supports client certificates, or if there's a recommended approach to incorporate them into the load testing process?

**Comment by SteveDesmond-ca** [comment]
Client certificates are not currently supported, but feel free to create an issue requesting this!
